src/main/scala/nl/knaw/dans/easy/fedora2vault/DDM.scala

```diff
@@ -74,6 +74,7 @@ object DDM extends DebugEnhancedLogging {
        { emd.getEmdRelation.getRelationMap.asScala.map { case (key, values) => values.asScala.map(toRelationXml(key, _)) } }
        { emd.getEmdContributor.getDcContributor.asScala.map(bs => <dc:contributor>{ bs.getValue }</dc:contributor>) }
        { emd.getEmdContributor.getEasContributor.asScala.map(author => <dcx-dai:contributorDetails>{ toXml(author)} </dcx-dai:contributorDetails>) }
+       { emd.getEmdRights.getTermsRightsHolder.asScala.map(bs => <dct:rightsHolder>{ bs.toString }</dct:rightsHolder>) }
```

Comment:
The new deposit form puts the rightsholder twice into the EMD. converting it back to DDM duplicates it again causing a broken test

https://github.com/DANS-KNAW/easy-fedora2vault/blob/8fffde4a8cf31a9375462fa3add85a5578647461/src/test/resources/sample-foxml/DepositApi.xml#L276-L297